tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -126,6 +127,28 @@ Delete entity tree
     ...    {"@topic-id":"device/child2//","@parent":"device/main//","@type":"child-device"}
     ...    ${entities}
 
+Entity twin update apis
+    ${entity}=    Execute Command
+    ...    curl -X PUT http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window -H 'Content-Type: application/json' -d '5'
+    ${entity}=    Evaluate    ${entity}
+    Should Be Equal    ${entity["maintenance_window"]}    ${5}
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_window
+    ...    message_contains=5
+
+    ${get}=    Execute Command
+    ...    curl http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window
+    Should Be Equal    ${get}    5
+
+    ${timestamp}=    Get Unix Timestamp
+    ${entity}=    Execute Command
+    ...    curl -X DELETE http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window
+    ${entity}=    Evaluate    ${entity}
+    Should Not Contain    ${entity}    maintenance_window
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_window
+    ...    date_from=${timestamp}
```

Comment:
It would be better to check if the retained message has been cleared rather than just checking if any message has been received (as it could also be a value update).

Reply:
Resolved by f837935c